README.adoc

```diff
@@ -29,6 +29,8 @@ buildPlugin()
   execute the steps against in parallel
 * `jenkinsVersions`: (default: `[null]`) - a matrix of Jenkins baseline versions to build/test against in parallel (null means default,
   only available for Maven projects)
+* `allowEmptyTestResults` (default: `false`) - Do not fail the build if the test results are missing.
+   It may be used for components, which have no tests within the repo
```

Comment:
Delete comma. You are saying something grammatically valid but quite different than what I think you meant: “components have no tests, so you may use this flag for them” → “if you have a component which has no tests, then you may use this flag for it” (subtle point of English punctuation; I know at least Czech lacks this rule)